### DISCUSSION TITLE
Preset for Segmind-Vega and Segmind-VegaRT

### DISCUSSION THREAD

**Discussion by Irmagon** [lllyasviel/Fooocus#1888] [Show and tell]
### The [Segmind-Vega](https://huggingface.co/segmind/Segmind-Vega) Model is a compact version with a remarkable 70% reduction in size compared to the Base SDXL Model.
Designed for efficiency, this model offers an impressive 100% speedup, making it suitable for real-time applications and scenarios where rapid image generation is essential.

![2024-01-12_02-18-26_5050](https://github.com/lllyasviel/Fooocus/assets/36661892/dfc85afe-2a99-42ef-b9c3-ce5d5864f46c)

```
{
    "default_model": "segmind-vega.safetensors",
    "default_loras": [["None",1.0],["None",1.0],["None",1.0],["None",1.0],["None",1.0]],
    "default_image_number": 1,
    "default_cfg_scale": 6.0,
    "default_sample_sharpness": 2.0,
    "default_sampler": "dpmpp_2m_sde_gpu",
    "default_scheduler": "karras",
    "default_performance": "Speed",
    "default_overwrite_step": 25,
    "default_prompt": "backlight, wilderness woman hunting in jungle hiding behind leaves, face paintings closeup face portrait, detailed eyes, nature documentary, dry skin, fuzzy skin, lens flare",
    "default_prompt_negative": "(deformed, distorted, disfigured:1.3), poorly drawn, bad anatomy, wrong anatomy, extra limb, missing limb, floating limbs, (mutated hands and fingers:1.4), disconnected limbs, mutation, mutated, ugly, disgusting, blurry, amputation, double middle finger, (thumbless, six-fingered:1.2)",
    "default_styles": ["Fooocus V2"],
    "default_aspect_ratio": "1024*1024",
    "checkpoint_downloads": {"segmind-vega.safetensors": "https://huggingface.co/segmind/Segmind-Vega/resolve/main/segmind-vega.safetensors?download=true"},
    "embeddings_downloads": {},
    "lora_downloads": {}
}
```

### [Segmind-VegaRT](https://huggingface.co/segmind/Segmind-VegaRT) a distilled consistency adapter for [Segmind-Vega](https://huggingface.co/segmind/Segmind-Vega) 
_That allows to reduce the number of inference steps to only between 2 - 8 steps._

This model is the first base model showing real-time capabilities at higher image resolutions, but has its own limitations:
1. The model is good at close up portrait images of humans but tends to do poorly on full body images.
2. Full body images may show deformed limbs and faces.
3. This model is an LCM-LoRA model, so negative prompt and guidance scale parameters would not be applicable.

![2024-01-12_01-59-54_7311](https://github.com/lllyasviel/Fooocus/assets/36661892/29db400e-27a2-46b6-bec3-8772daa9dd16)

```
{
    "default_model": "segmind-vega.safetensors",
    "default_loras": [["pytorch_lora_weights_vega.safetensors",1.0],["None",1.0],["None",1.0],["None",1.0],["None",1.0]],
    "default_advanced_checkbox": true,
    "default_image_number": 1,
    "default_performance": "Extreme Speed",
    "default_overwrite_step": 4,
    "default_prompt": "backlight, wilderness woman hunting in jungle hiding behind leaves, face paintings closeup face portrait, detailed eyes, nature documentary, dry skin, fuzzy skin, lens flare",
    "default_styles": ["Fooocus V2"],
    "default_aspect_ratio": "1024*1024",
    "checkpoint_downloads": {"segmind-vega.safetensors": "https://huggingface.co/segmind/Segmind-Vega/resolve/main/segmind-vega.safetensors?download=true"},
    "embeddings_downloads": {},
    "lora_downloads": {"pytorch_lora_weights_vega.safetensors": "https://huggingface.co/segmind/Segmind-VegaRT/resolve/main/pytorch_lora_weights.safetensors?download=true"}
}
```

**Comment by poor7** [comment]
How to install this in Colab?

**Comment by DavidDragonsage** [comment]
@Irmagon Thank you for the preset, that's a great way to distribute the model. I will definitely be checking out Vega.
